precompiles/ArbOwner.go

```diff
@@ -88,11 +88,27 @@ func (con ArbOwner) GetNetworkFeeAccount(c ctx, evm mech) (addr, error) {
 	return c.State.NetworkFeeAccount()
 }
 
+// Gets the infrastructure fee collector
+func (con ArbOwner) GetInfraFeeAccount(c ctx, evm mech) (addr, error) {
+	if c.State.FormatVersion() < 5 {
+		return common.Address{}, errors.New("")
```

Comment:
Is there a reason to keep this method when we have the one in `ArbOwnerPublic`?

Reply:
This follows the way it was done for `GetNetworkFeeAccount`, which is defined in both ArbOwner and ArbOwnerPublic.